### ISSUE TITLE
ataqc module fails

### ISSUE THREAD

**Issue by nicolerg** [ENCODE-DCC/atac-seq-pipeline#66]
After implementing the change in #64, `call-ataqc` fails with the following error:

```
Traceback (most recent call last):
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/bin/encode_ataqc.py", line 355, in <module>
    ataqc()
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/bin/encode_ataqc.py", line 186, in ataqc
    read_len)
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/bin/run_ataqc.py", line 438, in make_tss_plot
    processes=processes, stranded=True)
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/lib/python2.7/site-packages/metaseq/_genomic_signal.py", line 122, in array
    chunksize=chunksize, **kwargs)
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/lib/python2.7/site-packages/metaseq/array_helpers.py", line 383, in _array_parallel
    itertools.repeat(kwargs)))
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/lib/python2.7/multiprocessing/pool.py", line 253, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/users/nicolerg/anaconda2/envs/encode-atac-seq-pipeline/lib/python2.7/multiprocessing/pool.py", line 572, in get
    raise self._value
ValueError: invalid reference `chr1`
```

This might be related to non-standard chromosome names. I am using the `"atac.keep_irregular_chr_in_bfilt_peak" : false` option. 

**OS/Platform and dependencies**

- OS or Platform: Ubuntu 16.04.4 (durga)
- Cromwell/dxWDL version: cromwell 34-unknown-SNAP
- Conda version: conda 4.5.11

[debug_65.tar.gz](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/2654479/debug_65.tar.gz)

**Comment by nicolerg** [comment]
The tar file should be named debug_66.tar.gz

**Comment by leepc12** [comment]
Can you upload your genome TSV file and two files (`tss_enrich` and `chrsz`) in it?

**Comment by nicolerg** [comment]
The genome is the hard-masked version from the most recent Ensembl rat build: ftp://ftp.ensembl.org/pub/release-94/fasta/rattus_norvegicus/dna/

[tss.rn6.bed.gz](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/2659392/tss.rn6.bed.gz)
[rn6_masked.chrom.sizes.txt](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/2659393/rn6_masked.chrom.sizes.txt)
[rn6_masked.txt](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/2659394/rn6_masked.txt)

**Comment by leepc12** [comment]
This will be fixed in the next release v1.1.4. It will be released soon (this week).

**Comment by leepc12** [comment]
Closing this. Duplicate of #69.
